src/platforms/android/migration.mdx

```diff
@@ -7,8 +7,57 @@ redirect_from:
 
 ## Migrating from `io.sentry:sentry` `4.3.0` to `io.sentry:sentry` `5.0.0`
 
+If you're using the [Android Native Development Kit](/platforms/android/using-ndk/) support and you've set the `android:extractNativeLibs="true"` meta-data in the `AndroidManifest` file or `android.bundle.enableUncompressedNativeLibs=false` in the `gradle.properties` file, you don't have to anymore, you may remove them.
+Now Sentry is able to symbolicate events with the default value of [extractNativeLibs](https://developer.android.com/studio/releases/gradle-plugin#extractNativeLibs) and [android.bundle.enableUncompressedNativeLibs](https://developer.android.com/studio/releases/gradle-plugin#behavior-changes)
```

Comment:
@Swatinem review this part please, I guess we don't need to explain the reason, or do we?